Assets/MixedRealityToolkit.Tests/PlayModeTests/FocusProviderRaycastTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// Unity doesn't include the required assemblies (i.e. the ones below).
+// Given that the .NET backend is deprecated by Unity at this point it's we have
+// to work around this on our end.
+using Microsoft.MixedReality.Toolkit.Input;
+using NUnit.Framework;
+using System.Collections;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class FocusProviderRaycastTests
+    {
+        private GameObject raycastTestPrefabInstance = null;
+        private TestPointer pointer = null;
+        IMixedRealityFocusProvider focusProvider = null;
+
+        [UnityTest]
+        public IEnumerator TestRaycastProxies()
```

Comment:
Document the general test setup and workflow, e.g. test proxies are used to setup test cases and expected results, test below runs through them.